carservice/src/main/java/com/senlainc/warsaw/tyurin/controller/CraftsmanController.java

```diff
@@ -0,0 +1,74 @@
+package com.senlainc.warsaw.tyurin.controller;
+
+import com.senlainc.warsaw.tyurin.dto.CraftsmanDto;
+import com.senlainc.warsaw.tyurin.entity.Craftsman;
+import com.senlainc.warsaw.tyurin.exception.CriteriaNotFoundException;
+import com.senlainc.warsaw.tyurin.service.ICraftsmanService;
+import com.senlainc.warsaw.tyurin.util.builder.CraftsmanBuilder;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("craftsmen")
+public class CraftsmanController {
+
+    private final CraftsmanBuilder craftsmanBuilder;
+    private ICraftsmanService craftsmanService;
+
+    public CraftsmanController(CraftsmanBuilder craftsmanBuilder,
+                               ICraftsmanService craftsmanService) {
+        this.craftsmanBuilder = craftsmanBuilder;
+        this.craftsmanService = craftsmanService;
+    }
+
+    @ResponseBody
```

Comment:
Read about RestController, it already have ResponseBody inside, so this is redundant